src/main/java/com/onarandombox/MultiverseNetherPortals/listeners/MVNPEntityListener.java

```diff
@@ -316,11 +316,10 @@ public void onEntityPortal(EntityPortalEvent event) {
             return;
         }
 
-        // Some shortcuts for later
         Entity entity = event.getEntity();
 
-        Location fromLocation = this.locationManipulation.getBlockLocation(event.getFrom());
-        Location originalToLocation = this.locationManipulation.getBlockLocation(event.getTo());
+        Location fromLocation = event.getFrom();
+        Location originalToLocation = event.getTo();
```

Comment:
Why change this?

Reply:
Bcu the location can be decimal like 0.5. There isn't any reason for us to round it.